public/js/chess.js

```diff
@@ -17,6 +17,13 @@ const checkMove = ( event ) => {
   }
 }
 
+const reset = () => {
```

Comment:
The feels like a missing abstraction to me, which jumped out at me because of the change from `const` to `let`, above.  It seems to me that this code is working around behaviors that should be contained by/implemented in the `GameManager` class itself.  For example, isn't `reset`ting something a `GameManager` should be able to manage?  Why would we ever need more than one `GameManager`?